runtime/bindings-generator.cs

```diff
@@ -1915,6 +1915,18 @@ static IEnumerable<FunctionData> GetFunctionData ()
 				}
 			);
 
+			data.Add (
+				new FunctionData {
+					Comment = " // Matrix4 func (/* UIInterfaceOrientation */ nint)",
+					Prefix = "simd__",
+					Variants = Variants.All,
+					ReturnType = Types.Matrix4f,
+					Parameters = new ParameterData[] {
+						new ParameterData { TypeData = Types.NInt },
```

Comment:
@spouliot I can't get the previous declaration to fail if I use NInt, so I'm not sure why it failed. Also if any native type (NInt/NUInt/NFloat) is used in the signature, the binding generator will automatically generate both a 32-bit and 64-bit version of the functions as well, so it should have worked.